search/management/commands/update_index.py

```diff
@@ -0,0 +1,107 @@
+"""Management command to index reddit content"""
+from django.core.management.base import BaseCommand, CommandError
+
+from open_discussions.utils import now_in_utc
+from search.tasks import start_update_index
+from search.constants import VALID_OBJECT_TYPES, RESOURCE_FILE_TYPE
+from course_catalog.constants import PlatformType
+
+valid_object_types = list(VALID_OBJECT_TYPES)
+valid_object_types.append(RESOURCE_FILE_TYPE)
+
+
+class Command(BaseCommand):
+    """Indexes elasticsearch content"""
+
+    help = "Update elasticsearch index"
+
+    def add_arguments(self, parser):
+        allowed_course_platforms = [
+            PlatformType.ocw.value,
+            PlatformType.mitx.value,
+            PlatformType.micromasters.value,
+            PlatformType.bootcamps.value,
+            PlatformType.xpro.value,
+            PlatformType.oll.value,
+            PlatformType.see.value,
+            PlatformType.mitpe.value,
+            PlatformType.csail.value,
+        ]
```

Comment:
```suggestion
        allowed_course_platforms = [pt.value for pt in PlatformType]
```

Reply:
The array only includes course platform types: `youtube` and `podcast` are not included. But starting with all PlatformTypes then excluding is cleaner

Reply:
Oh, ok :+1: